Android/app/src/main/java/com/example/samplestickerapp/StickerApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.example.samplestickerapp;
```

Comment:
Please add the following copyright header:

/*
 * Copyright (c) WhatsApp Inc. and its affiliates.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree.
 */